snapcraft/internal/sources/__init__.py

```diff
@@ -87,10 +86,8 @@
 
 from snapcraft.internal import common
 from snapcraft import file_utils
-from snapcraft.internal.indicators import (
-    download_requests_stream,
-    download_urllib_source
-)
+from . import errors
+from . import _base
```

Comment:
I say don't even import base, it is not needed. Your internal package modules should import using relative paths